### DISCUSSION TITLE
Why are there few PAL games supported?

### DISCUSSION THREAD

**Discussion by Ryoma123456789** [garungorp/MouseInjectorDolphinDuck#6] [General]
I don't understand why many of the games compatible with this software are NTSC, what is the problem with PAL games? Why are there no European developers interested in the project, or don't they know anything about it?

**Comment by ZeptoBST** [comment]
PAL versions of games are typically regarded as inferior to NTSC because they run around 16% slower than NTSC (although it is possible to force a PAL game to run at NTSC speed in some emulators). As such many people tend to focus on NTSC versions instead.
